tests/test_manure_module/test_manure_treatments.py

```diff
@@ -2032,3 +2083,123 @@ def test_bound_sludge_accumulation_volume(mocker: MockFixture) -> None:
     # Assert
     assert patch_for_sludge_accumulation_volume_property.call_count == 2
     assert actual_bound_sludge_accumulation_volume == approx(expected_bound_sludge_accumulation_volume)
+
+
+# Test AnaerobicDigestion-specific methods
+# ========================================
+
+def test_calc_anaerobic_digestion_daily_output(mocker: MockFixture) -> None:
+    # Arrange
+    pass
+
+
+@pytest.mark.parametrize(
+        'total_daily_mass',
+        [
+            0.0,
+            100.0,
+        ])
+def test_calc_moisture_content(total_daily_mass: float) -> None:
+    """Unit test for _calc_moisture_content() in anaerobic_digestion.py."""
```

Comment:
I'd also suggest making these test docstrings a bit more descriptive. Rather than saying is a unit test, say what it is meant to check. For example:
```suggestion
    """check that moisture content is properly calculated by  _calc_moisture_content()."""
```
It's still 1 line but it tells the reader what you intended the test to do.

Reply:
I think it's not that important to have a descriptive docstring under the name of each unit test. Though, the two good places for that are when you have multiple edge cases inside the unit test and when you want to annotate each case in the parameterized list above the unit test. But I do get your point of the more descriptive the better as opposed to something generic.